net.xmind.XMind8.yml

```diff
@@ -0,0 +1,82 @@
+id: net.xmind.XMind8
+runtime: org.freedesktop.Platform
+runtime-version: '18.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk8
+tags:
+  - proprietary
+command: XMind
+separate-locales: false
+build-options:
+  no-debuginfo: true
+finish-args:
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --share=network
+  - --device=dri
+  - --filesystem=xdg-documents
+  - --filesystem=xdg-download
+  - --filesystem=xdg-pictures
+  - --filesystem=xdg-videos
+  - --filesystem=/run/media
+  - --filesystem=/media
+  - --env=GTK_IM_MODULE=xim
+cleanup:
+  - "*.a"
+  - "*.la"
+  - /include
+  - /lib/pkgconfig
+  - /lib/cmake
+  - "/cache"
+  - "/include"
+  - "/lib/pkgconfig"
+  - "/man"
+  - "/share/aclocal"
+  - "/share/devhelp"
+  - "/share/gir-1.0"
+  - "/share/gtk-doc"
+  - "/share/man"
+  - "/share/pkgconfig"
+  - "/share/vala"
+  - "/lib/systemd"
+modules:
+  - shared-modules/gtk2/gtk2.json
+  - shared-modules/lame/lame-3.99.5.json
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - mkdir /app/jre
+      - cp -a /usr/lib/sdk/openjdk8/jvm/java-8-openjdk/* /app/jre
+      - rm -rf /app/jre/src.zip /app/jre/include /app/jre/demo /app/jre/sample
+  - name: xmind
+    buildsystem: simple
+    build-commands:
+      - install -Dm755 XMind.sh /app/bin/XMind
+      - install -Dm755 apply_extra.sh /app/bin/apply_extra
+      - install -Dm644 ${FLATPAK_ID}.appdata.xml -t /app/share/appdata/
+      # Icons have be to extracted from extra data
+      - install -Dm644 ${FLATPAK_ID}.desktop -t /app/
+      - ln -s ../extra/fonts /app/share/fonts
+    sources:
+      - type: file
+        path: apply_extra.sh
+
+      - type: file
+        path: XMind.sh
+
+      - type: file
+        path: net.xmind.XMind8.appdata.xml
+
+      - type: file
+        path: net.xmind.XMind8.desktop
+
+      - type: extra-data
+        filename: xmind.zip
+        only-arches:
+          - 'x86_64'
+          - 'i386'
+        url: https://dl2.xmind.cn/xmind-8-update8-linux.zip
+        sha256: 164abcc27d124721d58d5bd8f8941686519203ce7580f48133e73544c34145d0
```

Comment:
Does the vendor also provide versioned tarball? It may be problematic to keep it possible to install as we still lack automation for that.

Reply:
`update8` is the version of tarball.